src/buildblock/Scanner.cxx

```diff
@@ -817,12 +861,12 @@ check_consistency() const
 	      this->get_name().c_str());
     else
       {
-	if ( get_num_detectors_per_ring() % get_num_transaxial_crystals_per_singles_unit() != 0)
+    if ( get_num_detectors_per_ring() % (get_num_transaxial_crystals_per_singles_unit() + get_num_virtual_transaxial_crystals_per_block()) != 0)
 	  { 
 	    warning("Scanner %s: inconsistent transaxial singles unit info:\n"
-		    "\tnum_detectors_per_ring %d should be a multiple of num_transaxial_crystals_per_singles_unit %d",
+            "\tnum_detectors_per_ring %d should be a multiple of num_transaxial_crystals_per_singles_unit + virtual %d",
```

Comment:
shouldn't be changed

Reply:
But this is what you have in mCT.
This is an important comment: If you don't include the virtual crystals in the total number of detectors per ring the unlisting fails. The sinogram is broken. I could send an image to show you.

Reply:
It has something to do with the inversion in the tangestial positions when the tang_pos is larger than the number of views. Which currently is calculated from the total number of detectors per ring.

Reply:
>  If you don't include the virtual crystals in the total number of detectors per ring the unlisting fails. The sinogram is broken.

of course. Siemens does all calculations/sinograms in terms of "total number". Nowhere in their files there are any "virtual" ones (even not in the norm, presumably because they do gap-filling)